vital/types/track.h

```diff
@@ -36,14 +36,14 @@
 #ifndef VITAL_TRACK_H_
 #define VITAL_TRACK_H_
 
-
-#include <vital/vital_export.h>
+#include <vital/attribute_set.h>
 #include <vital/vital_config.h>
+#include <vital/vital_export.h>
 #include <vital/vital_types.h>
 
-#include <vector>
-#include <set>
 #include <memory>
+#include <set>
+#include <vector>
```

Comment:
Dies this still need set and vector?

Reply:
Yes:
`vector` → history
`set` → frame IDs

Note that I didn't add these, I just reordered the includes to be in lexicographical order as they should be.

(It's possible we get them transitively, but even if so, I'm not particularly inclined to remove them as they are directly used.)